website/content/docs/ecs/compatibility.mdx

```diff
@@ -0,0 +1,25 @@
+---
+layout: docs
+page_title: Compatibility Matrix
+description: Compatibility Matrix for Consul ECS
+---
+
+# Compatibility Matrix for Consul on ECS
+
+For every release of Consul on ECS, `consul-ecs` binary and `consul-ecs` terraform module is updated. The terraform module and binary are tightly coupled. Ex: consul-ecs binary 0.4.1 must use consul-ecs terraform 0.4.1.
+
+## Supported Consul versions
+
+### Version 0.2.0 and above
```

Comment:
```suggestion
```
I don't think you need this subheading unless you were going to add a table for versions 0.1.X